### PR TITLE
Add Cian Issue#4 Contribution

### LINKED ISSUE #4: Add Cientera Rose
**Please add your name to the contribution-list.md**

### PR DESCRIPTION
**Pull request by Ciantera-Rose**
Add issue to contribution list

### CODE REVIEW

File: contribution_list.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # People who have helped!
 
 > Please add your name below to be added to the list of awesome people that have helped with this code.
+
+[Ciantera Rose](https://github.com/Ciantera-Rose/git-help-me)
```
Comment (gleekzorp) [inline contribution_list.md]: Please fix the url as it is incorrect.

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please see comments and fix url.  You can use your personal github url.

File: contribution_list.md (hunk 1 above)
Comment (Ciantera-Rose) [reply contribution_list.md]: Okay sure thing I'll update the url

File: contribution_list.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 > Please add your name below to be added to the list of awesome people that have helped with this code.
 
-[Ciantera Rose](https://github.com/Ciantera-Rose/git-help-me)
+[Ciantera Rose](https://github.com/Ciantera-Rose?tab=repositories)
```
Comment (gleekzorp) [inline contribution_list.md]: Please update with the following link.

https://github.com/Ciantera-Rose

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please update with correct link

**Review by gleekzorp — APPROVED** [review]
Awesome!

**Comment by Ciantera-Rose** [comment]
Resolve git url repo contribution

**Pull request merged**